docs/Algorithms.md

```diff
@@ -153,3 +153,188 @@ as developed for the GA4GH as well as Disease Codes (**DS-X**) from the [Human D
 		</tr>	
 	</tbody>
 </table>
+
+## Version 3
```

Comment:
Let's add some UI enhancements here. We should have v1, v2 collapsed and v3 expanded. See https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections for how to do that.

Also, let's move v3 to the top, v2 next, then v1 last.